5. Linked List/grouping/LinkedlistGroupingArnab.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * LinkedListGroupingArnab.java
+ * Print a Linked List with all even indices arranged first,and then the odd indices
+ * Description:-
+ * The approach is simple,just store the even and odd position values in 2 separate arraylist and then create a separate linked 
```

Comment:
See again in linked list this kind of solution is easy to come up with but don't you think perhaps you can do better? How about not creating an arraylist but doing operation in the same linked list or maybe create another linked list. Think!